common/save-game.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <vector>
+
+#include <save_game.pb.h>
+
+#include "filesystem.hpp"
+
+#include "tec-types.hpp"
+
+namespace tec {
+using uid = decltype(proto::User().id());
```

Comment:
I almost feel like this deserves a comment :)

Reply:
Can do!